tests/modules/atomicslotmigration.c

```diff
@@ -334,6 +356,9 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     if (RedisModule_CreateCommand(ctx, "asm.replicate_module_command", replicate_module_command, "", 0, 0, 0) == REDISMODULE_ERR)
         return REDISMODULE_ERR;
 
+    if (RedisModule_CreateCommand(ctx, "asm.replicate_crossslot_command", replicate_crossslot_command, "", 0, 0, 0) == REDISMODULE_ERR)
```

Comment:
btw please rename replicate -> propagate for this and the above command as you prefer that way :)